elastic-stack/elasticsearch/map.jinja

```diff
@@ -17,6 +17,7 @@
         'log_folder': '/var/log/elasticsearch/',
         'conf_folder': '/etc/elasticsearch/',
         'conf_file': '/etc/elasticsearch/elasticsearch.conf',
+        'java_home': '/usr/lib/jvm/java-1.8.0-openjdk-amd64'
```

Comment:
Do we want to hardcode the java version?

Reply:
I was wondering about that, but though it would be in line with the `pkg_deps` properties ([example](https://github.com/mitodl/elastic-stack-formula/blob/c13c61ac32c1196dbdd62c5213c716e9205f32fe/elastic-stack/elasticsearch/map.jinja#L23)) that call for Java 8.

Reply:
`java-1.8.0-openjdk-amd64` is a symlink to `java-8-openjdk-amd64` in the same directory, so maybe I should use the more general one (`java-8-openjdk-amd64`).

Reply:
Also, since this is the map file it's something we can override via pillar data

Reply:
I'm fine with either option